commands/project.go

```diff
@@ -0,0 +1,93 @@
+package commands
+
+//handle the project logic for rhmap.
+
```

Comment:
First thoughts. And this is up for debate, In my mind I had imagined a structure where this command would be part of the get package then we might create a delete package and a create package etc. Which would mean we would end up with 

```
rhm get projects
rhm delete project

```
etc